src/Styleguide/Components/Pagination.tsx

```diff
@@ -6,39 +6,88 @@ import { Responsive } from "Styleguide/Utils/Responsive"
 import { Arrow } from "Styleguide/Elements/Arrow"
 import { Flex } from "Styleguide/Elements/Flex"
 
-export class Pagination extends React.Component {
+interface PageCursor {
+  page: number
+  cursor: string
+  isCurrent: boolean
+}
+
+interface PaginationProps {
+  first?: PageCursor
+  last?: PageCursor
+  around: ReadonlyArray<PageCursor>
+  onClick?: (cursor: string) => void
+  onNext?: () => void
+  onPrev?: () => void
+}
+
+export class Pagination extends React.Component<PaginationProps> {
+  static defaultProps = {
+    onClick: cursor => {
+      // tslint:disable-next-line:no-console
+      console.log(`Clicked ${cursor}`)
+    },
+    onNext: () => {
+      // tslint:disable-next-line:no-console
+      console.log("Clicked onNext")
+    },
+    onPrev: () => {
+      // tslint:disable-next-line:no-console
+      console.log("Clicked onPrev")
+    },
+  }
```

Comment:
We should lean on VSCode / TypeScript to warn you about necessary things rather than possibly masking through default props. And not necessarily pass anything (logs are kind of noisy unless they're necessary for legit logging, but either way!),  just complete the contract -- could be something like `x => x`, etc.